wbooks/styles.js

```diff
@@ -3,6 +3,6 @@ import { StyleSheet } from 'react-native';
 export default StyleSheet.create({
     container: {
     height: '100%', 
-    backgroundColor: 'cyan'
+    backgroundColor: '#EAF6FA'
```

Comment:
remember you have a 'constants' folder. We should create now a `colors.js` file in it that will export all the colors of the app

Reply:
done